### ISSUE TITLE
Hide by keywords issue

### ISSUE THREAD

**Issue by blimeybloke** [njelich/LinkOff#46]
I am not sure if this is a bug or feature request.
I have added a group I follow to the hide by keywords and I still see all posts in this group.

I do not want to leave the group but I do not need to see 100 posts a day.

So either its a bug, or a feature to hide groups.

Cheers

**Comment by njelich** [comment]
I'll test. If there are not enough posts to scroll and generate new ones, nothing will be hidden until you scroll.

**Comment by blimeybloke** [comment]
Thanks!  I would also love to offer some suggestions on the Job section too.
A simple one may be to sort by newest.  Their job section is a complete disaster,

**Comment by valdoryu** [comment]
As of the refacto of the extension, I can't reproduce your issue with groups.

Your suggestion to sort jobs by newest would be nice, unfortunately Linkedin show jobs publication date only when focusing the job, making sorting very difficult/impossible to accomplish

**Comment by valdoryu** [comment]
@blimeybloke  Did you have time to try again ?

**Comment by blimeybloke** [comment]
> [@blimeybloke](https://github.com/blimeybloke) Did you have time to try again ?

Yes, all is working.  No more political stuff in my feed.  Thank you.
